consensus/applytransaction.go

```diff
@@ -13,13 +13,12 @@ func (s *State) applySiacoinInputs(bn *blockNode, t Transaction) {
 			}
 		}
 
-		scod := SiacoinOutputDiff{
-			New:           false,
+		bn.siacoinOutputDiffs = append(bn.siacoinOutputDiffs, SiacoinOutputDiff{
```

Comment:
my habit is to not declare new variables if it can be avoided (except in multi-stage computations). Makes it more clear that you aren't going to use `scod` later. Just like:
```go
encBlock := encoding.Marshal(b)
if len(encBlock) > MaxSize {
```
vs.
```go
if len(encoding.Marshal(b)) > MaxSize {
```
I will admit it's a bit uglier in this case though.